tests/libc/CMakeLists.txt

```diff
@@ -17,7 +18,21 @@ if (UNIX)
 	add_subdirectory(enc)
 endif()
 
+string(TOUPPER ${CMAKE_BUILD_TYPE} BUILD_TYPE_UPPER)
 foreach(testcase ${alltests})
+    if (CMAKE_C_COMPILER_ID MATCHES Clang)
+	if (BUILD_TYPE_UPPER MATCHES DEBUG)
+            list(FIND clang_debug ${testcase} output)
```

Comment:
We're requiring a new enough version of CMake that you can just use `if(<variable|string> IN_LIST <variable>)` instead of `list(FIND ... ) if (...)`, so:

```
if (${testcase} IN_LIST clang_debug)
  continue ()
endif ()
```

Reply:
Thanks. Hadn't thought about that.